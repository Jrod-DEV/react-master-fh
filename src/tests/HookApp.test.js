import React from 'react';
import { shallow } from 'enzyme';
import { HookApp } from '../HookApp';

describe('Tests on <HookApp /> component', () => {
  test('should show correctly', () => {

    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
    
  });
});
