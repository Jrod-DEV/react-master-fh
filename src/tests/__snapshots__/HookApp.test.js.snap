// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests on <HookApp /> component should show correctly 1`] = `ShallowWrapper {}`;
